### PR TITLE
Laravel 7 support

### PR DESCRIPTION
**Pull request by RichardDominik**

### CODE REVIEW

File: tests/Feature/AdminUsers/AdminUserCrudGeneratorWithCustomControllerNameTest.php [hunk 1]
```diff
@@ -38,6 +39,7 @@ public function admin_user_controller_name_can_be_namespaced(): void
 use Brackets\AdminAuth\Services\ActivationService;
 use Brackets\AdminListing\Facades\AdminListing;
 use Exception;
+use Illuminate\Support\Facades\Auth;
```
Comment (dejwCake) [inline tests/Feature/AdminUsers/AdminUserCrudGeneratorWithCustomControllerNameTest.php]: Why is this required?

Comment (RichardDominik) [reply tests/Feature/AdminUsers/AdminUserCrudGeneratorWithCustomControllerNameTest.php]: This facade is used for the impersonal login feature https://github.com/BRACKETS-by-TRIAD/admin-generator/blob/laravel-7/resources/views/templates/admin-user/controller.blade.php#L298
